site/content/gitops-quickstart/01-apply-gitops.md

```diff
@@ -0,0 +1,260 @@
+```
+
+Copy the lines starting with `ssh-rsa` and add it as a deploy key, to
+e.g. Github. There you can easily do this in the
+`Settings > Deploy keys > Add deploy key`. Just make sure you check
+`Allow write access` as well.
+
+The next time Flux syncs from git, it will start updating the cluster
+and actively deploying. If you use the [`flux-get-started`
+repo](https://github.com/fluxcd/flux-get-started) from above, here's
+what you will see in your cluster:
+
+```console
+$ kubectl get pods -n demo
+NAME                       READY   STATUS    RESTARTS   AGE
+podinfo-5f4bd464b4-4vf7k   1/1     Running   0          58m
+podinfo-5f4bd464b4-hkgzt   1/1     Running   0          58m
+```
+
+Remember that you can further tweak the installation of Flux as
+discussed in our [`install flux` docs](/usage/experimental/gitops-flux/).
+
+### Handcrafting your configuration
+
+In this second step we will use `eksctl generate config`, so you can
+easily handcraft your cluster configuration and maintain it in Git, just
+like you would for the workloads you are running.
+
+During the previous command (`eksctl install flux`), we instructed Flux
+to watch a config repository. This is where the workloads are defined.
+Now we will add the cluster config as well.
+
+In this example we will run:
+
+```console
+EKSCTL_EXPERIMENTAL=true eksctl generate profile \
+        --name wonderful-wardrobe-1565767990 \
+        --git-url git@github.com:happy-gopher/eks-gitops-example.git \
+        --profile-path ~/dev/flux-get-started/cluster-config
+```
+
+Let's break this down here. `eksctl generate profile` at the very
+least wants:
+
+- `--name`: the name of the cluster - check `eksctl get cluster`
+  to see what the name of yours is
+- `--git-url`: the Git URL of the base config of your cluster
+- `--profile-path`: a local path: this is an empty new directory
+  (here `cluster-config`) you create in your local checkout of
+  the config repository, which we used in the previous command
+
+The base config can be something you and your organisation
+tailored to your needs, but can be something like our [EKS
+GitOps example][eks-gitops-example] as well. It is meant to be
+a starting point for clusters you can iterate over.
+
+[eks-gitops-example]: https://github.com/weaveworks/eks-gitops-example
+
+Check `XXX` for a description of all the flags and options -
+blocked on <https://github.com/weaveworks/eksctl/pull/1140>.
+
+So after all this preface, what happens when we run the command?
+`eksctl` will check out the base-config (here we use a fork of the
+GitOps example config) in an empty sub-directory (`cluster-config`)
+of our local checkout of our `flux-get-started` fork.
```

Comment:
They are not forks, I think you mean local copies of the repos.

Reply:
If they start off of e.g `weaveworks/eksctl-gitops-example`, they won't be able to commit to it, so they'll need a fork, right? I'll still write it up a bit more clearly.

Reply:
Not really. For their gitops repo they need an empty repo. And it shouldn't be a fork of anything else. Most likely this is for an internal environment in their companies and they should use their own repos, probably private, with a name that makes sense like "production-kubernetes-infra" for example.

Regarding the base repo, if a user wants to create a profile repository that others can use as their quick starts, they could theoretically use a fork ok eks-gitops-example, but they can start from an empty repo.

So I think we should not encourage the use of forks (also we've discussed this a few times within the team, feel free to ping us on slack and we can explain better :) )